speeduino/speeduino.ino

```diff
@@ -334,7 +328,7 @@ void loop(void)
       #endif
 
       //Check for any outstanding EEPROM writes.
-      if( (isEepromWritePending() == true) && (serialReceivePending == false) && (micros() > deferEEPROMWritesUntil)) { writeAllConfig(); } 
+      if( (isEepromWritePending() == true) && (serialReceivePending == false) && (micros() - lastEEPROMDeferTime > EEPROM_DEFER_DELAY)) { writeAllConfig(); } 
```

Comment:
```suggestion
      if( (isEepromWritePending() == true) && (serialReceivePending == false) && ((unsigned long)(micros() - lastEEPROMDeferTime) > EEPROM_DEFER_DELAY)) { writeAllConfig(); } 
```

Reply:
The cast probably shouldn't be needed as `micros()` is an unsigned long already, but the brackets are better here.

Reply:
I've added the brackets.